File: app/src/main/java/com/dinodevs/greatfitwatchface/theme/bin/Date.java

package com.dinodevs.greatfitwatchface.theme.bin;

import java.util.HashMap;
import java.util.Map;

public class Date {

    private MonthAndDay monthAndDay;
    private WeekDay weekDay;
    private Map<String, Object> additionalProperties = new HashMap<String, Object>();

    public MonthAndDay getMonthAndDay() {
        return monthAndDay;
    }

    public void setMonthAndDay(MonthAndDay monthAndDay) {
        this.monthAndDay = monthAndDay;
    }

    public WeekDay getWeekDay() {
        return weekDay;
    }

    public void setWeekDay(WeekDay weekDay) {
        this.weekDay = weekDay;
    }

    public Map<String, Object> getAdditionalProperties() {
        return this.additionalProperties;
    }

    public void setAdditionalProperty(String name, Object value) {
        this.additionalProperties.put(name, value);
    }

}

File: app/src/main/java/com/dinodevs/greatfitwatchface/theme/bin/Date.kt
package com.dinodevs.greatfitwatchface.theme.bin

import java.util.*

class Date {
    var monthAndDay: MonthAndDay? = null
    var weekDay: WeekDay? = null
    private val additionalProperties: MutableMap<String, Any> = HashMap()

    fun getAdditionalProperties(): Map<String, Any> {
        return additionalProperties
    }

    fun setAdditionalProperty(name: String, value: Any) {
        additionalProperties[name] = value
    }
}